lib/core.js

```diff
@@ -53,184 +59,187 @@ var core = module.exports = {
 	 *   app_secret: 'xxxxx',						//app secret - required, config use init method or pass through here
 	 *   v: '2.0',									//api version - optional, default is 2.0
 	 *   sign_method: 'md5'							//sign method - optional, default is md5, now only support md5
-	 *   
+	 *
 	 *   // method specific arguments
 	 *
 	 * }
 	 * </pre>
 	 *
 	 * @param {Function}
 	 *            callback function with parameters to get the result
-	 * 
+	 *
 	 * @returns {Array[search...]}
 	 */
 	call: function (httpArgs, args, callback) {
-		//compatiple with call (args, callback) signature
-		if (arguments.length == 2) {
-			callback = args;	
-			args = httpArgs;
-			httpArgs = {};
+		// compatiple with call (args, callback) signature
+		if (arguments.length === 2) {
+			callback = args
+			args = httpArgs
+			httpArgs = {}
 		}
 
-		args.method = args.method;
-		args.timestamp = args.timestamp || moment(new Date()).format('YYYY-MM-DD HH:mm:ss');
-		args.format = args.format || 'json';
-		args.app_key = args.app_key || cfg.app_key;
-		args.app_secret = args.app_secret || cfg.app_secret;
-		args.v = args.v || '2.0';
-		args.sign_method = args.sign_method || 'md5';
-		args.session = args.session || cfg.session;
+		// args.method = args.method
+		args.timestamp = args.timestamp || moment(new Date()).format('YYYY-MM-DD HH:mm:ss')
```

Comment:
is moment only being used for datestamp formatting or do we have anyother evaluations as well?